build/test-aot.ps1

```diff
@@ -16,7 +16,7 @@ foreach ($line in $($publishOutput -split "`r`n"))
 }
 
 Write-Host "Actual warning count is: ", $actualWarningCount
-$expectedWarningCount = 63
+$expectedWarningCount = 61
```

Comment:
if the count is lower, that is okay. we don't want it going up. I assume some dependencies were updated.